tuned_lens/model_surgery.py

```diff
@@ -211,8 +211,8 @@ def permute_layers_(model: th.nn.Module, indices: list[int]):
 
 @contextmanager
 def replace_layers(
-    model: T, indices: list[int], replacements: list[th.nn.Module]
-) -> Generator[T, None, None]:
+    model: Model, indices: list[int], replacements: list[th.nn.Module]
+) -> Generator[None, None, None]:
```

Comment:
The model is modified in place right? That's why returning it is not needed.

Reply:
Why do other functions (e.g. `delete_layers`, `permute_layers`, `assign_key_path`) still yield the modified model?

Reply:
Ah good point I'll change those as well!

Reply:
FWIW the reason I initially made it return the model was to mirror PyTorch in-place functions and methods which, for better or worse, also return the tensor. But I don't actually care as long as we're consistent.